### DISCUSSION TITLE
Zod to prop types converter.

### DISCUSSION THREAD

**Discussion by louiss0** [dwightjack/vue-types#438] [Ideas]
I think there should be a way to convert Zod schemas to Vue prop types.

## Motivation 

When writing schemas you will often use a validation library like `zod` `yup` or even valibot. 
A component's props in most situations will have a schema that is similar to the Zod schema 
you are using for requests. If there could be a way to convert `zod` schemas to vue prop types then 
that would be awesome.

**Comment by dwightjack** [comment]
@louiss0 Thanks for contributing. I am not familiar with zod objects, but feel free to submit a PR with a demo convert function 😊

**Reply by louiss0** [reply]
Zod is a schema validator just like Yup. It makes sure that values are the correct type.  Click [here](https://zod.dev) to check it out.
